src/systems/sound-effects-system.js

```diff
@@ -138,7 +138,7 @@ export class SoundEffectsSystem {
     const audioBuffer = this.sounds.get(sound);
     if (!audioBuffer) return null;
 
-    const disablePositionalAudio = isSafari() || window.APP.store.state.preferences.audioOutputMode === "audio";
+    const disablePositionalAudio = isSafari() || window.APP.store.state.preferences.disableLeftRightPanning === true;
```

Comment:
Can `window.APP.store.state.preferences.disableLeftRightPanning === true` be reduced to `window.APP.store.state.preferences.disableLeftRightPanning` here?

Reply:
yep, I think so, we are only using it for the ternary. Updated.